assets/samples/cloudNativeFullStack.yaml

```diff
@@ -88,8 +92,10 @@ spec:
           key: node-role.kubernetes.io/control-plane
           operator: Exists
 
-      # Optional: resource settings for OneAgent container. Consumption of the OneAgent heavily depends
-      # on the workload to monitor; please adjust values accordingly.
+      # Optional: Adds resource settings for OneAgent container
+      # Consumption of the OneAgent heavily depends on the workload to monitor
+      # The values should be adjusted according to the workload
+      # An example to start with is provided below
```

Comment:
Would remove the last sentence, it clear that the following lines are examples + we did not write it to the other example either